lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,11 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.nil? or comments.last.nil?
```

Comment:
Agreed, we should make comments always return an `Array`, then the check here is simpler.